test/correctness/load_library.cpp

```diff
@@ -0,0 +1,76 @@
+#include <stdio.h>
+#include "Halide.h"
+
+using namespace Halide;
+
+// This test exercises the ability to override halide_get_library_symbol (etc)
+// when using JIT code; to do so, it compiles & calls a simple pipeline
+// using an OpenCL schedule, since that is known to use these calls
+// in a (reasonably) well-defined way and is unlikely to change a great deal 
+// in the near future; additionally, it doesn't require a particular
+// feature in LLVM (unlike, say, Hexagon).
+
+namespace {
+
+void my_error_handler(void* u, const char *msg) {
+    if (!strstr(msg, "OpenCL API not found")) {
+        fprintf(stderr, "Saw unexpected error: %s\n", msg);
+        exit(-1);
+    }
+    printf("Saw expected error: %s\n", msg);
```

Comment:
Wouldn't this test "pass" if the custom JIT handlers were broken, and the OpenCL runtime was also missing?